packages/ui/atomic/create-atomic-result-component/index.js

```diff
@@ -120,11 +136,27 @@ const ensureComponentValidity = (tag) => {
   }
 };
 
+const ensureDirectoryValidity = () => {
```

Comment:
Can't the error and check be reused for the 2 workflows?

Reply:
This is what I was originally planning on doing, but that would require moving multiple things to a dedicated package (e.g., `coveo/create-atomic-commons`). There's an issue to accomplish exactly that in progress right now:
https://coveord.atlassian.net/browse/CDX-1428

That being said, perhaps I should hold off on merging this until that other issue is done, then it _**can**_ be shared 😅